src/utils/setup-status.ts

```diff
@@ -31,7 +39,12 @@ export async function createSetupStatusTracker(
 		timeout = setTimeout(() => void startChecking(), 1_000);
 	};
 
-	await startChecking();
+	await timeTracker.run(
+		"setup-status-tracker.checkIsSetupRequired",
```

Comment:
nit: to have the same format as in other instances:
```suggestion
		"setup-status.checkIsSetupRequired",
```